ios/NativeBridge.swift

```diff
@@ -99,6 +106,16 @@ public class NativeBridge: NSObject {
                 }
             }
     }
+
+    @objc public func resumeWebAuth(url: String, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
+        if let value = URL(string: url) {
+            if (!WebAuthentication.resume(with: value)) {
+                reject("ERROR_PARSING_URL", "The callback url \(url) is invalid", nil)
+            }
+        } else {
+            reject("ERROR_PARSING_URL", "The callback url \(url) is invalid", nil)
+        }
```

Comment:
Maybe rework this to avoid those nested ifs and duplicated rejections? Both `URL(string: url)` initializing to `nil` and `WebAuthentication.resume(with: value)` mean that the URL is invalid.